export.py

```diff
@@ -2740,11 +2740,16 @@ def export_display(ri, rpass, scene):
         ri.DisplayChannel(source_type + ' %s' % (channel_name), params)
 
 
+    #Flushes any output file names left over from previous render sessions
+    while rm.output_files_index > -1:
```

Comment:
Right but if the same rpass is around for multiple frames...

On Fri, Jun 10, 2016 at 2:48 PM, Jon D <notifications@github.com> wrote:

> In export.py
> <https://github.com/bsavery/PRMan-for-Blender/pull/280#discussion_r66685800>
> :
>
> > @@ -2740,11 +2740,16 @@ def export_display(ri, rpass, scene):
> >          ri.DisplayChannel(source_type + ' %s' % (channel_name), params)
> >
> >
> > +    #Flushes any output file names left over from previous render sessions
> > +    while rm.output_files_index > -1:
>
> It does that at the beginning of RPass in the engine module
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/bsavery/PRMan-for-Blender/pull/280/files/dda4821cb2cb7557ffc777bb0b74e84586357eb4#r66685800>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABzffAm9Ixi2E6Gp1xQWJnYpEmOYxdJVks5qKdvDgaJpZM4IzU-0>
> .
>



-- 
brian.savery@gmail.com
508-274-8700